express/scripts/templates.js

```diff
@@ -9,14 +9,26 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+import {
+  getHelixEnv,
+  // eslint-disable-next-line import/no-unresolved
+} from './scripts.js';
+
 async function fetchPageContent(path) {
   if (!(window.templates && window.templates.data)) {
     window.templates = {};
     const resp = await fetch('/express/templates/content.json?sheet=seo-templates');
     window.templates.data = resp.ok ? (await resp.json()).data : [];
   }
 
-  return window.templates.data.find((p) => p.path === path && p.live !== 'N');
+  const page = window.templates.data.find((p) => p.path === path);
+  const env = getHelixEnv();
+
+  if (env && env.name === 'stage') {
+    return page ?? null;
```

Comment:
I think `||` would be safer in terms of browser (older safari) support:
```suggestion
    return page || null;
```